public/pages/workflow_detail/workflow_inputs/processor_inputs/input_transform_modal.tsx

```diff
@@ -116,24 +124,105 @@ export function InputTransformModal(props: InputTransformModalProps) {
           </EuiFlexItem>
           <EuiFlexItem>
             <>
-              <EuiText>Define transform with JSONPath</EuiText>
+              <EuiText>Define transform</EuiText>
+              <EuiText size="s" color="subdued">
+                {`Dot notation is used by default. To explicitly use JSONPath, please ensure to prepend with the
+                root object selector "${JSONPATH_ROOT_SELECTOR}"`}
+              </EuiText>
               <EuiSpacer size="s" />
-              <EuiCodeEditor
-                mode="json"
-                theme="textmate"
-                value={`TODO`}
-                readOnly={false}
-                setOptions={{
-                  fontSize: '12px',
-                  autoScrollEditorIntoView: true,
-                }}
-                tabSize={2}
+              <MapField
+                field={props.inputMapField}
+                fieldPath={props.inputMapFieldPath}
+                label="Input map"
+                helpText={`An array specifying how to map fields from the ingested document to the model’s input.`}
+                helpLink={
+                  'https://opensearch.org/docs/latest/ingest-pipelines/processors/ml-inference/#configuration-parameters'
```

Comment:
We should have the links in a common file or the constant file

Reply:
Good point, will move in the next PR